### ISSUE TITLE
Dragging without selecting

### ISSUE THREAD

**Issue by ReyMagos** [files-community/Files#3451]
**Problem**: In a default explorer item can be moved simply by dragging this to destination place. But in Files item can be dragged only after it was selected. It's a not good thing and it takes many time.
**Solution**: Item must be able to be moved without pre-selection.

**Comment by XPoppyX** [comment]
Duplicate of #673
